dockers/Screenshotter/screenshotter.js

```diff
@@ -128,17 +130,53 @@ if (seleniumURL) {
     console.log("Selenium driver in local session");
 }
 
-if (!katexURL) {
-    katexURL = "http://" + katexIP + ":" + opts.katexPort + "/";
-}
 var toStrip = "http://localhost:7936/"; // remove this from testcase URLs
 
+process.nextTick(startServer);
+var attempts = 0;
+
+//////////////////////////////////////////////////////////////////////
+// Start up development server
+
+var devServer = null;
+var minPort = 32768;
+var maxPort = 61000;
+
+function startServer() {
+    if (katexURL || katexPort) {
+        process.nextTick(tryConnect);
```

Comment:
I'll have to start using this.  It's way nicer than the `timeout(callback, 0)` trick that I'm used to.